### PR TITLE
Bug #201589 Learners journey - Identified status is missing in users journey

### PR DESCRIPTION
**Pull request by Tusharmahajan12**
Updated audit logs and when the registration of beneficiarires the status will be identified and it will show in journey
https://tracker.tekdi.net/issues/201589?tab=time_entries

### CODE REVIEW

File: src/src/modules/auth/auth.service.ts [hunk 1]
```diff
@@ -811,6 +838,17 @@ export class AuthService {
                 user_id
                 type
               }
+			  program_beneficiaries {
```
Comment (yogesh-pawar-2023) [inline src/src/modules/auth/auth.service.ts:841]: format document

Comment (Tusharmahajan12) [reply src/src/modules/auth/auth.service.ts]: done changes please check

File: src/src/beneficiaries/beneficiaries.service.ts [hunk 2]
```diff
@@ -187,6 +189,10 @@ export class BeneficiariesService {
 						id: 'aadhaar_verification_mode',
 						title: 'Aadhaar Verification Mode',
 					},
+					{
+						id: 'aadhaar_verification_mode',
+						title: 'Aadhaar Verification Mode',
+					},
```
Comment (parthgandhitekdi) [inline src/src/beneficiaries/beneficiaries.service.ts:195]: Why this code block added? It is already there

File: src/src/beneficiaries/beneficiaries.service.ts [hunk 3]
```diff
@@ -207,14 +213,20 @@ export class BeneficiariesService {
...
 					: 'no';
 				dataObject['aadhaar_verification_mode'] =
 					data?.aadhaar_verification_mode;
 				records.push(dataObject);
+				dataObject['aadhar_no'] = data?.aadhar_no;
+				dataObject['aadhar_verified'] = data?.aadhar_verified
+					? data?.aadhar_verified
+					: 'no';
+				dataObject['aadhaar_verification_mode'] =
+					data?.aadhaar_verification_mode;
```
Comment (parthgandhitekdi) [inline src/src/beneficiaries/beneficiaries.service.ts:229]: This code block is duplicated. Why it's added?

File: src/src/modules/auth/auth.controller.ts [hunk 4]
```diff
@@ -92,6 +92,7 @@ export class AuthController {
 	public async register(
 		@Body() body: RegisterDTO,
 		@Res() response: Response,
+		@Req() request: any,
```
Comment (parthgandhitekdi) [inline src/src/modules/auth/auth.controller.ts:95]: What is the need of request here?

File: src/src/modules/auth/auth.service.ts [hunk 5]
```diff
@@ -825,6 +852,17 @@ export class AuthService {
...
```
Comment (parthgandhitekdi) [inline src/src/modules/auth/auth.service.ts:855]: format document

**Review by parthgandhitekdi — CHANGES_REQUESTED** [review]

File: src/src/beneficiaries/beneficiaries.service.ts (hunk 2 above)
Comment (Tusharmahajan12) [reply src/src/beneficiaries/beneficiaries.service.ts]: removed

File: src/src/modules/auth/auth.controller.ts (hunk 4 above)
Comment (Tusharmahajan12) [reply src/src/modules/auth/auth.controller.ts]: removed

File: src/src/modules/auth/auth.service.ts (hunk 5 above)
Comment (Tusharmahajan12) [reply src/src/modules/auth/auth.service.ts]: formatted

**Review by parthgandhitekdi — APPROVED** [review]

**Pull request merged**